drivers/gnss/Kconfig.generic

```diff
@@ -0,0 +1,26 @@
+# Copyright 2023 Google LLC
```

Comment:
Are these GNSS changes somehow intertwined with the modem driver changes? I was not expecting to see these in here.

Reply:
Yeah they are to some extent as I encountered some merge conflicts without taking in those GNSS changes.

Reply:
I think that the modem driver stuff is likely OK, as we are the only user for many of those, however, I am a bit worried about these GNSS changes. Ping @trantanen.

Reply:
Sorry, wrong Tommi. Ping @tokangas.

Reply:
The Zephyr GNSS API is already present in sdk-zephyr, but we don't use it currently. So, these commits should be fine.